Description: automake 1.11 needs -Wno-obsolete due to ansi2knr (for autoreconf)
 or fails with message 'automatic de-ANSI-fication support is deprecated'.
Forwarded: not-needed
Last-Update: 2014-06-30
Author: Mauricio Faria de Oliveira <mauricfo@linux.vnet.ibm.com>

Index: libjpeg6b-6b1/configure.ac
===================================================================
--- libjpeg6b-6b1.orig/configure.ac	2010-05-15 16:28:51.000000000 -0300
+++ libjpeg6b-6b1/configure.ac	2014-06-30 18:07:36.000000000 -0300
@@ -21,7 +21,7 @@
 
 # Initialize Automake
 # Don't require all the GNU mandated files
-AM_INIT_AUTOMAKE([-Wall -Werror ansi2knr no-dist foreign])
+AM_INIT_AUTOMAKE([-Wall -Werror -Wno-obsolete ansi2knr no-dist foreign])
 
 # Make --enable-silent-rules the default.
 # To get verbose build output you may configure
